ansible_community_meetings.ics

```diff
@@ -5,8 +5,6 @@ BEGIN:VEVENT
 SUMMARY:Ansible Security Automation working group
 DTSTART;VALUE=DATE-TIME:20210405T150000Z
 DURATION:PT1H
-DTSTAMP;VALUE=DATE-TIME:20191126T180022Z
```

Comment:
These lines seem to have been added in the previous commit, though don't appear in previous versions.

Reply:
Maybe another version of the tool that generates the file?

Reply:
I updated yaml2ics and I didn't get them. Maybe a mac vs Linux thing?